### PR TITLE
[bitnami/mongodb] Issue 2096. Encode @ : in username / password when connect to mongo

### PR DESCRIPTION
**Pull request by lff0305**
If `@` or `:` in username / password, the metric exporter will not connect to mongodb, because
`mongo://username:password@localhost` will not be splitted correctly.

So here the `@` and `:` is replaced to `%40` and `%3A` (url-encode) by `sed` to fix the issue.

### CODE REVIEW

File: bitnami/mongodb/templates/statefulset-primary-rs.yaml [hunk 1]
```diff
@@ -229,7 +229,7 @@ spec:
               secretKeyRef:
                 name: {{ if .Values.existingSecret }}{{ .Values.existingSecret }}{{- else }}{{ template "mongodb.fullname" . }}{{- end }}
                 key: mongodb-root-password
-          command: [ 'sh', '-c', '/bin/mongodb_exporter --mongodb.uri mongodb://root:${MONGODB_ROOT_PASSWORD}@localhost:{{ .Values.service.port }}/admin {{ .Values.metrics.extraArgs }}' ]
+          command: [ 'sh', '-c', '/bin/mongodb_exporter --mongodb.uri mongodb://root:`echo $MONGODB_ROOT_PASSWORD | sed -r "s/@/%40/g"`@localhost:{{ .Values.service.port }}/admin {{ .Values.metrics.extraArgs }}' ]
```
Comment (andresbono) [inline bitnami/mongodb/templates/statefulset-primary-rs.yaml]: This one is not making the substitutions for `:`

**Comment by andresbono** [comment]
Please, bump the version of the chart in the Charts.yaml file.

File: bitnami/mongodb/templates/statefulset-primary-rs.yaml (hunk 1 above)
Comment (lff0305) [reply bitnami/mongodb/templates/statefulset-primary-rs.yaml]: Missed this one. Will update.

**Comment by andresbono** [comment]
Thanks for your contribution. We are now going to test the PR and merge it.

**Pull request merged**